config/routes.php

```diff
@@ -15,4 +16,13 @@
         ->options([
             'priority' => 1,
         ]);
+
+    $routes->add(Route::STORYBLOK_CONTENT_TYPE, '/{!slug}')
+        ->controller(ContentTypeController::class)
+        ->requirements([
+            'slug' => ContentTypeController::PATH_PATTERN,
+        ])
+        ->options([
+            'priority' => -10000,
+        ]);
```

Comment:
I'd put this route in a recipe

Reply:
This is already in a recipe

Reply:
but if you don't use `symfony/flex`, you need to configure it manually :-(